t3.dart

```diff
@@ -0,0 +1,13 @@
+void switchInt3() {
+  const int A = 1;
+  int v = 5;
+
+  switch (v) {
+    case 1:
+    case 2:
+    case A:
+    case 2:
+    case 3:
+    default:
+  }
```

Comment:
The value of the case clause ('A') is equal to the value of an earlier case clause ('1').

```suggestion
```